draft-dekater-scion-controlplane.md

```diff
@@ -1356,7 +1357,7 @@ All of these are manageable on a present day small server or desktop machine.
 For much larger, more highly connected ASes, the path-discovery tasks of the control service can be distributed over many instances in order to increase the PCB throughput.
 
 
-On a cold start of the network, full connectivity is obtained after a number of propagation steps corresponding to the diameter of the network. Assuming a network diameter of 6, this corresponds to roughly 3 minutes on average.
+On a cold start of the network, full connectivity is obtained at worst after a number of propagation steps corresponding to the diameter of the network, unless ASes propagate beacons more frequently at bootstrapping. Assuming a network diameter of 6, this corresponds to roughly 3 minutes on average.
```

Comment:
And again. Here I think it would be enough to mention the worst case and not bother with "unless...".